gemfiles/523.gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+gemspec :path=>"../"
+gem "minitest", "~> 5.19.0"
```

Comment:
ah this was wrong, making a fixup

Reply:
Will fix in 2 minutes

Reply:
already got a PR ... trying to look into the CI failure next